ras/sosreport.py.data/options.yaml

```diff
@@ -5,3 +5,12 @@ list     : ['-o dlm -k dlm.lockdump', '-k filesys.dumpe2fs=on',
 case_id  : "test123"
 file_list: ['proc/device-tree/', 'hostname', 'mount', 'netstat', 
             'ps', 'sos_commands', 'sos_logs', 'sos_reports']
+
+dir: '/mnt'
+filesystem: !mux
+    ext4:
+        fs: 'ext4'
+    xfs:
+        fs: 'xfs'
+    ext3:
+        fs: 'ext3'
```

Comment:
Can you remove ext3 and instead add btrfs? Also make sure to add a check for mkfs.btrfs in the code. If this command is missing the test case should not error-out.

Reply:
will remove ext3 and as discussed over slack will not add btrfs as it anyways not supported on Rhel. And also addressed the above comments